assets/css/plugin-check-admin.css

```diff
@@ -0,0 +1,52 @@
+/* Responsive table */
+@media only screen and (max-width: 40em) {
```

Comment:
Why specifically 40em?

Reply:
Any size that is mobile. We can do 400px. But what would be your suggestion?

Reply:
WordPress admin uses `@media screen and (max-width: 782px)` elsewhere for example.

Reply:
let's do that.